contributing_to_docs/doc_guidelines.adoc

```diff
@@ -649,6 +649,21 @@ concept is being referred to. For example:
 - Operator
 - DaemonSet (and not "daemon set", "daemonset", or "Daemonset")
 
+[[operator-name-capitalization]]
+==== Operator Name Capitalization
+
+An Operator's full name should be a proper noun, with each word initially
+capitalized. If it includes a product name, defer the product's capitalization
+style guidelines, if available (for example, lowercase etcd).
```

Comment:
To be consistent with the previous section, I'd s/, if available (for example, lowercase etcd). Examples:/. For example: